src/Rings/mpoly-ideals.jl

```diff
@@ -878,9 +900,10 @@ end
 Base.:in(f::MPolyElem, I::MPolyIdeal) = ideal_membership(f,I)
 #######################################################
 @doc Markdown.doc"""
-    radical_membership(f::T, I::MPolyIdeal) where T <: MPolyElem
+    radical_membership(f::T, I::MPolyIdeal{T}) where T <: MPolyElem
    
 Return `true` if `f` is contained in the radical of `I`, `false` otherwise.
+Alternatively, use `inradical(f, I)`.
```

Comment:
Why? What's the point in having two names for this?

Reply:
Because @fieker asked for it.